roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -40,6 +40,9 @@
     openshift_hosted_registry_images: "{{ openshift.hosted.registry.registryurl | default('openshift3/ose-${component}:${version}')}}"
     openshift_hosted_registry_volumes: []
     openshift_hosted_registry_env_vars: {}
+    openshift_hosted_registry_route_certificates: "{{ openshift.hosted.registry.route.certificates | default(false) }}"
+    openshift_hosted_registry_route_host: "{{ openshift.hosted.registry.route.host | default(false) }}"
```

Comment:
@detiber, @mtnbikenc 

This is a good question.  Let's see if I can explain:
The host for the route needs to be different depending on the type of route that we are creating.
docker_registry_route_hostname is a product of https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_hosted/tasks/registry/secure.yml#L4

and the variable above( openshift_hosted_registry_route_host ) we want to take precedence if it is set:
`    host: "{{ openshift_hosted_registry_route_host | default(docker_registry_route_hostname) }}"`